config/esp32/components/chip/Kconfig

```diff
@@ -350,7 +350,7 @@ menu "CHIP Device Layer"
         config MAX_EVENT_QUEUE_SIZE
             int "Max Event Queue Size"
             range 0 65535
-            default 100
+            default 20
```

Comment:
@gjc13  Is this the event queue for the CHIP task? Also, is 20 the optimized value?

Reply:
Yes, this is the event queue in the device layer. I'll try 5-10 as well.